languages/python/sqlalchemy-oso/tests/test_flask.py

```diff
@@ -29,18 +29,26 @@ def ctx(flask_app):
         yield ctx
 
 
+# Global because test_flask_model wants to enforce authorization on a model
+# that's created after initialization of the AuthorizedSQLAlchemy instance.
+checked_permissions = {}
```

Comment:
This is fairly self-explanatory but definitely a wowhack (lowercase) and worth calling out.

I know @leina05 mentioned wanting to remove the `flask_sqlalchemy` integration (`AuthorizedSQLAlchemy` in `languages/python/sqlalchemy-oso/sqlalchemy_oso/flask.py`) entirely, and I would be happy to do that as a follow-up to this PR.